python/graphstorm/model/edge_decoder.py

```diff
@@ -132,6 +132,8 @@ class DenseBiDecoder(GSEdgeDecoder):
     norm: str
         Normalization methods. Not used, but reserved for complex DenseBiDecoder child class
         implementation. Default: None.
+    use_bias: bool
```

Comment:
Can you also add the ``.. versionchanged:: 0.4.0`` annotation for both edge decoders and node decoders, since we change the API.
Here is an example:
https://github.com/awslabs/graphstorm/blob/ddae1f3b70c1186af840008955f44dd2089e63e5/python/graphstorm/inference/mt_infer.py#L113-L115